pset-functions.js

```diff
@@ -174,7 +301,18 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
-
+function isPalindrome(){
+
+    let palRev = pal.split("").reverse().join("");
+
+    if(pal === palRev){
+        console.log(`${pal} is a palindrome!`)
+    } else{
+        console.log(`${pal} is NOT a palindrome!`)
+    }
+}
+let pal = "noosh"
+isPalindrome()
```

Comment:
@jl1982 Great work here with this solution. Nice job using method chaining to reverse the string and to compare that with the original string. One suggestion when you call the function with the variable` pal` as a parameter.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L242)